xarray/core/indexing.py

```diff
@@ -5,7 +5,9 @@
 from contextlib import suppress
 from datetime import timedelta
 from typing import Any, Callable, Iterable, List, Sequence, Tuple, Union
+from distutils.version import LooseVersion
 
+from dask import __version__ as dask_version
```

Comment:
Dask is a soft dependency. So, we probably need a conditional import here


```python
try:
    import dask
except ModuleNotFoundError:
    ...

```

Reply:
Would putting the import statement it in the class DaskIndexingAdapter work, since it's would only ever be executed if dask is available? Or is importing in classes not typically done in xarray?